pom.xml

```diff
@@ -158,7 +158,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>2.22.1</version>
+                    <version>3.0.0-M5</version>
```

Comment:
Why this change? This is not the scope of this issue. Please roll this change back, and apply it in a new PR with the corresponding issue.

Reply:
Yeah, I think it would be better to have these plugin updates handled in a separate pull request.